config/routes.js

```diff
@@ -0,0 +1,20 @@
+
+const MarkerAPIController = require("../src/Controllers/MarkerAPIController");
+const AddNewWalkController = require("../src/Controllers/AddNewWalkController");
+const ErrorController = require("../src/Controllers/ErrorController")
+
+function routes(app){
+    app.get('/markers', MarkerAPIController.getAllStartMarkersController);
+    app.post('/markers', ErrorController);
+    app.put('/markers', ErrorController)
+    app.delete('/markers', ErrorController)
+
+    app.get('/walks', ErrorController);
+    app.post('/walks', AddNewWalkController);
+    app.put('/walks', ErrorController);
+    app.delete('/walks', ErrorController);
+
+    app.get('/markers/:id', MarkerAPIController.getDogWalkInfoController);
```

Comment:
Just for organisational purposes, can you move this route so it's directly under the get to /markers?